views-and-forms-java/src/main/java/views/and/forms/java/controllers/SurveyController.java

```diff
@@ -0,0 +1,55 @@
+package views.and.forms.java.controllers;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Consumes;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.views.View;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import views.and.forms.java.model.FormData;
+import views.and.forms.java.model.FruitChoices;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Controller("/")
+public class SurveyController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SurveyController.class);
+
+    @Get
+    @View("home")
+    public FruitChoices home() {
+
+        LOG.info("Sending home page " );
```

Comment:
Should be wrapped with `if (LOG.isInfoEnabled())`

Reply:
Not needed these days, plus there is no string concatenation. An `if` clause here (and anywhere in reality these days, if parameterized messages are used) is not needed :wink: